.travis.yml

```diff
@@ -41,4 +41,4 @@ script:
   - travis_run_tests
 
 after_success:
-  - travis_after_tests_successs
+  - travis_after_tests_success
```

Comment:
Just to bring attention to this - we will remove before merge, it's just so we can identify the coverage gaps while working. A rebase onto current 9 contained a not fun amount of conflict work, so this seemed like a good interim solution.